bulkcopy_test.go

```diff
@@ -81,6 +82,10 @@ func TestBulkcopy(t *testing.T) {
 		{"test_binary", []byte("1"), nil},
 		{"test_binary_16", bin, nil},
 		{"test_intvarchar", 1234, "1234"},
+		{"test_int64nvarchar", int64(123456), "123456"},
+		{"test_int32nvarchar", int32(12345), "12345"},
+		{"test_int16nvarchar", int16(1234), "1234"},
```

Comment:
Would it be better if these test values are values that don't fit in the domain of the next smallest type?